### PR TITLE
feat: SfAccordionItems can be opened programmaticaly

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/stories/releases/v0.10.9/v0.10.9.stories.mdx [hunk 1]
```diff
@@ -7,16 +7,14 @@ import { Meta } from "@storybook/addon-docs/blocks";
...
-- SfSelect cannot click on label to open the dropdown
 - SfButton - disabled state made more inclusive
 
 ## 🐛 Fixes
 
+- SfSelect cannot click on label to open the dropdown
 - SfRadio: fixed to work on Safari 11 and older
 - SfProductCardHorizontal - access to quantity during adding to cart
 - SfRange - removed optional chaining
+- SfAccordion - SfAccodrionItems can be opened programmatically
```
Comment (Baroshem) [inline packages/vue/src/stories/releases/v0.10.9/v0.10.9.stories.mdx:18]: This should be a feature IMO.

**Review by Baroshem — APPROVED** [review]
Small change in the changelog and you are ready to go!

File: packages/vue/src/stories/releases/v0.10.9/v0.10.9.stories.mdx (hunk 1 above)
Comment (Baroshem) [reply packages/vue/src/stories/releases/v0.10.9/v0.10.9.stories.mdx:18]: Also SfAccordionItems not SfAccodrionItems ;)

**Review by AdamPawlinski — APPROVED** [review]
It works great. One thing worth to discuss is do we need firstOpen prop in this component. The same result can be achieved with open prop. But it is discussion for future probably as it would be breaking change.

**Pull request merged**